doc/schemas/lightning-listsendpays.json

```diff
@@ -144,7 +144,7 @@
             "amount_msat": {
               "type": "msat",
               "description": [
-                "The amount delivered to destination (if known)."
+                "The amount delivered to destination (if known). This field might not be present if the payment is still pending or if the amount delivered to the destination is not known."
```

Comment:
No, it means that the caller used "sendonion" so we don't know how much actually got delivered.